charts/sentry-apps/values.yaml

```
@@ -0,0 +1,78 @@
+spec:
+  metadata:
+    # spec.metadata.name -- name for the ArgoCD application
+    name: sentry
+    # spec.metadata.namespace -- namespace for the ArgoCD application
+    namespace: argocd
+  destination:
+    # spec.destination.namespace -- namespace for Sentry
+    namespace: sentry
+  # spec.project -- project to deploy the ArgoCD application to
+  project: default
+  source:
+    # spec.source.chart -- name of the Chart for Sentry
+    chart: sentry
+    # spec.source.path -- path of the Chart for Sentry when using Git repository
+    path: ""
+    # spec.source.repoURL -- Chart museum to get Sentry
+    repoURL: "https://sentry-kubernetes.github.io/charts"
+    # spec.source.targetRevision -- revision of the chart to use for Sentry
+    targetRevision: 4.7.2
```

If you use `default .Chart.AppVersion` you can get rid of it in values.
```suggestion
    # spec.source.targetRevision -- Override the revision of the chart to use for Sentry
    # @default -- Chart appVersion
    targetRevision: ""
```